dags/hivemind_etl_helpers/notion_ingestion_etl.py

```diff
@@ -0,0 +1,67 @@
+import os
+
+from dotenv import load_dotenv
+from llama_index.core import MockEmbedding
+from llama_index.core.ingestion import (DocstoreStrategy, IngestionCache,
+                                        IngestionPipeline)
+from llama_index.core.node_parser import SemanticSplitterNodeParser
+from llama_index.storage.docstore.postgres import PostgresDocumentStore
+from llama_index.storage.kvstore.redis import RedisKVStore as RedisCache
+from llama_index.vector_stores.postgres import PGVectorStore
+from tc_hivemind_backend.embeddings.cohere import CohereEmbedding
+
+from dags.hivemind_etl_helpers.src.db.gdrive.db_utils import setup_db
+
+
+class NotionIngestionPipeline:
+    def __init__(self, community_id: str, testing: bool = False):
+        load_dotenv()
+        host = os.getenv("POSTGRES_HOST")
+        password = os.getenv("POSTGRES_PASS")
+        port = os.getenv("POSTGRES_PORT")
+        user = os.getenv("POSTGRES_USER")
+        self.redis_host = os.getenv("REDIS_HOST")
+        self.redis_port = os.getenv("REDIS_PORT")
+
+        self.table_name = "notion"
+        self.dbname = f"community_{community_id}"
+        self.db_config = {
```

Comment:
please consider updating the block that reads the postgres credentials to use this function

To load the credentials you can use this
<https://github.com/TogetherCrew/hivemind-backend/blob/main/tc_hivemind_backend/db/credentials.py>

Installing the `tc_hivemind_backend` you can use the load credentials via the function
```python
from tc_hivemind_backend.db.credentials import load_postgres_credentials
```